test/rules_test.js

```diff
@@ -217,6 +217,61 @@ describe('rules', () => {
         ['x^(|-(x + 1)|)', 'x^(x + 1)'],
     ])
 
+    suite('simplify fraction', rules.SIMPLIFY_FRACTION, [
+        ['1/3', '1 / 3'],
+        ['2/6', '1 / 3'],
+        ['15/24', '5 / 8']
+    ])
+
+    suite('cancel exponent', rules.CANCEL_EXPONENT, [
+        ['nthRoot(x^2, 4)', 'nthRoot(x^1, 2)'],
+        ['nthRoot(y^3, 4)', 'nthRoot(y^3, 4)'],
+        ['nthRoot(a^15, 24)', 'nthRoot(a^5, 8)'],
+        ['nthRoot(b^4, 2)', 'b^2'],
+        ['nthRoot(c^8, 3)', 'nthRoot(c^8, 3)'],
+        ['nthRoot(d^10, 10)', 'd^1'],
+        ['nthRoot(x^2)', 'x^1'],
+    ])
+
+    suite('combine under root', rules.COMBINE_UNDER_ROOT, [
+        ['nthRoot(2, 2) * nthRoot(3, 2)', 'nthRoot(2 * 3, 2)'],
+        ['nthRoot(4, 3) * nthRoot(3, 3)', 'nthRoot(4 * 3, 3)']
+    ])
```

Comment:
Can you add tests cases for n-ary instances as well as cases where the radicand and the index are identifiers?